qbittorrent/start.sh

```diff
@@ -100,13 +100,22 @@ echo "[INFO] Starting qBittorrent daemon..." | ts '%Y-%m-%d %H:%M:%.S'
 /bin/bash /etc/qbittorrent/qbittorrent.init start &
 chmod -R 755 /config/qBittorrent
 
-# Wait a second for it to start up and get the process id
-sleep 1
-qbittorrentpid=$(pgrep -o -x qbittorrent-nox) 
-echo "[INFO] qBittorrent PID: $qbittorrentpid" | ts '%Y-%m-%d %H:%M:%.S'
+# wait for the qbittorrent.init script to finish and grab the qbittorrent pid
+# from the file created by the start script
+wait $!
```

Comment:
If it's safe to move the `chmod` call on line 101 to execute after qbittorrent starts, then the `wait` call here as well as the `&` on line 100 could be removed. I wasn't sure so I left it intact.